lang/fr.json

```diff
@@ -1,68 +1,68 @@
-  "scale_of_suffering.wild_mammals": "10-100 mammifères sauvages",
-  "scale_of_suffering.reptiles_and_amphibians": "10-10,000 reptiles et amphibiens",
-  "scale_of_suffering.fish": "1,000 - 100,000 poisson",
-  "scale_of_suffering.earthworms": "10,000 - 10⁶ vers de terre",
-  "scale_of_suffering.terrestrial_arthropods": "600,000 - 7*10⁸ arthropodes terrestres",
-  "scale_of_suffering.insects": "600,000 - 7*10⁸ insectes",
-  "scale_of_suffering.in_existence": "…en existence à tout moment.",
+  "scale_of_suffering.farmed_fish": "... sans compter les poissons d'élevage, dont le nombre est estimé à environ 10-13 par personne.",
+  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en entre :",
+  "scale_of_suffering.wild_birds": "10 et 50 oiseaux sauvages",
+  "scale_of_suffering.wild_mammals": "10 et 100 mammifères sauvages",
+  "scale_of_suffering.reptiles_and_amphibians": "10 et 10 000 reptiles et amphibiens",
+  "scale_of_suffering.fish": "1000 et 100 000 poissons",
+  "scale_of_suffering.earthworms": "10 000 et 10⁶ vers de terre",
+  "scale_of_suffering.terrestrial_arthropods": "600 000 et 7x10⁸ arthropodes terrestres",
+  "scale_of_suffering.insects": "600 000 - 7x10⁸ insectes",
+  "scale_of_suffering.in_existence": "… en existence à tout moment.",
   "scale_of_suffering.conclusion": "Cela signifie que la grande majorité des individus sont des animaux sauvages. L'échelle est vraiment stupéfiante.",
-  "types_of_suffering.id": "types-of-suffering",
+  "types_of_suffering.id": "types-de-souffrance",
   "types_of_suffering.name": "Types de souffrance",
   "types_of_suffering.heading": "<0>Pire encore,</0><1>Les animaux sauvages subissent de terribles souffrances:</1>",
   "types_of_suffering.hint": "Cliquez sur un article ci-dessus pour en savoir plus.",
   "types_of_suffering.dialog_learn_more": "En savoir plus",
-  "types_of_suffering.dialog_go_back": "Retourner",
-  "types_of_suffering.accidents.description": "Les animaux sont souvent victimes d'accidents et de blessures dans la nature. Ils peuvent être blessés par des prédateurs ou lors de conflits intraspécifiques pour le territoire ou l'accouplement ; brûlés par des incendies ou gelés par des gelées soudaines ; piégés par des terrains difficiles tels que des mares de boue ou des lacs gelés, et confrontés à une mort douloureuse et prolongée ; ou ils peuvent simplement avoir un accident dans le cours normal de leur vie, tout comme les êtres humains. Cependant, contrairement aux humains, les animaux sauvages disposent rarement d'une aide efficace en cas d'accident ou de blessure. Ils se retrouvent presque sans défense face à des animaux plus forts, à des conditions climatiques extrêmes et à des pièges naturels. Néanmoins, les humains parviennent parfois à sauver des animaux blessés ou piégés, même dans des circonstances difficiles.",
+  "types_of_suffering.dialog_go_back": "Revenir",
```

Comment:
I don't have the context where it's being used. Would "Retour" not be better suited?

Reply:
Yes, absolutely! Thanks